poolrpc/trader.proto

```diff
@@ -871,7 +871,11 @@ message LeaseDurationRequest {
 }
 
 message LeaseDurationResponse {
-    map<uint32, bool> lease_durations = 1;
+    /*
+    The set of lease durations the market is currently accepting and the state
+    the duration buckets currently are in.
+    */
+    map<uint32, DurationBucketState> lease_duration_buckets = 1;
```

Comment:
this breaks clients of the trader service.

Reply:
You're right, maybe someone is already using the RPC or REST interface. Deprecated the legacy format instead of replacing it.